### PR TITLE
ref(hc): OrganizationMember.user_is_active backfill

### PR DESCRIPTION
**Pull request by corps**
Backfills new column that allows User is active queries on OrganizationMember

### CODE REVIEW

File: src/sentry/migrations/0471_backfill_organization_member_is_active.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_om_is_active(apps, schema_editor):
+    OrganizationMember = apps.get_model("sentry", "OrganizationMember")
+    for om in RangeQuerySetWrapperWithProgressBar(OrganizationMember.objects.all()):
+        # Default is true, so update only on False.
+        if not om.user.is_active:
+            om.update(user_is_active=False)
```
Comment (wedamija) [inline src/sentry/migrations/0471_backfill_organization_member_is_active.py:14]: `.update` doesn't work in migrations, you'll need to set the field and use `.save(update_fields=["user_is_active"])`

Comment (corps) [reply src/sentry/migrations/0471_backfill_organization_member_is_active.py:14]: Thank you dude!!

**Review by wedamija — APPROVED** [review]

File: src/sentry/migrations/0471_backfill_organization_member_is_active.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+from sentry.new_migrations.migrations import CheckedMigration
+from sentry.utils.query import RangeQuerySetWrapperWithProgressBar
+
+
+def backfill_om_is_active(apps, schema_editor):
+    OrganizationMember = apps.get_model("sentry", "OrganizationMember")
+    for om in RangeQuerySetWrapperWithProgressBar(OrganizationMember.objects.all()):
+        # Default is true, so update only on False.
+        if not om.user.is_active:
```
Comment (markstory) [inline src/sentry/migrations/0471_backfill_organization_member_is_active.py:13]: You might want to pre-load users with select_related('user') so that we don't to N*2 queries here.

File: src/sentry/migrations/0472_delete_past_organization_mappings.py [hunk 3]
```diff
@@ -28,7 +28,7 @@ class Migration(CheckedMigration):
     is_dangerous = False
 
     dependencies = [
-        ("sentry", "0471_dashboard_widget_description"),
+        ("sentry", "0472_delete_past_organization_mappings"),
```
Comment (dashed) [inline src/sentry/migrations/0472_delete_past_organization_mappings.py:31]: Why was this changed? 👀

**Review by dashed — APPROVED** [review]
